src/server.py

```diff
@@ -297,11 +300,13 @@ def terminate_sound_for_sender(self, uuid_or_event_id, connection, sender,
                                is set to 0.
         """
         assert not (uuid_or_event_id in self.registry.sounds and
-                    uuid_or_event_id in self.registry.uuids_by_event_id)
+                    (uuid_or_event_id in
+                        self.registry.sound_events.get_event_ids()))
```

Comment:
Optional: I think it would be nice to have a method on `SoundEventsRegistry` to check if an event ID exists, to clean this up and use it below as well. E.g. `SoundEventsRegistry.has_sound_event_id()`.